js/app/modules/arrangement/tiledGrid.js

```diff
@@ -28,8 +28,8 @@ const TiledGrid = new Module.Class({
         this.get_children().forEach(this.remove, this);
         // The card to be packed is already in this array:
         let cards = this.get_cards();
-
-        let columns = Math.ceil(this.get_card_count() / 2);
+        let count = this.get_card_count();
+        let columns = count == 2 ? 2 : Math.ceil(this.get_card_count() / 2);
```

Comment:
While you're at it, I would probably refactor this so that 3 cards make 3 columns too.